charts/redpanda/values.yaml

```diff
@@ -271,6 +271,8 @@ monitoring:
   enabled: false
   scrapeInterval: 30s
   labels: {}
+  enableHttp2: true
```

Comment:
please update the values.schema.json file with these new entries.

Reply:
Is there a plugin which generates the schema? I tried one but it changes most of the current file

Reply:
No, not that I know of, You can manually add it, i can help if you need it.